core/domain/stats_services.py

```diff
@@ -515,7 +515,8 @@ def get_exploration_stats_by_id(exp_id, exp_version):
     return exploration_stats
 
 
-def get_multiple_exploration_stats_by_version(exp_id, version_numbers):
+def get_multiple_exploration_stats_by_version(
```

Comment:
Maybe keep this as it was before? (Don't see any change besides the line break.)

Reply:
Ah yes, done!